org.scala-ide.sdt.core/src/org/scalaide/extensions/autoedits/CloseMatchingPair.scala

```diff
@@ -0,0 +1,99 @@
+package org.scalaide.extensions
+package autoedits
+
+import org.eclipse.jface.text.IRegion
+import org.scalaide.util.eclipse.RegionUtils._
+
+/** Functionality for all auto edits that should handle matching pairs. */
+trait CloseMatchingPair extends AutoEdit {
+
+  private val openingMap = Map('{' -> '}', '[' -> ']', '(' -> ')', '<' -> '>', '"' -> '"')
```

Comment:
I added them nevertheless, no one needs to enable them. ;) Maybe some hardcore XML coders out there.